script/upgradeUnipoolGIVpower.s.sol

```diff
@@ -5,26 +5,20 @@ import 'forge-std/console.sol';
 import 'forge-std/Script.sol';
 import '@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol';
 import '@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol';
-import '../contracts/UnipoolGIVpower.sol';
-import '../contracts/interfaces/IDistro.sol';
+import './hardcode-upgrade/UnipoolGIVpower.sol';
+import './hardcode-upgrade/interfaces/IDistro.sol';
 
-
-contract UpgradeUnipoolGIVpower is Script {
-/// @notice Op Sepolia addresses
-address proxyAdminOpSepoliaAddress = 0x3b197F5cDa3516bD49e193df6F1273f3f16d414a;
-address unipoolProxyOpSepoliaAddress = 0xE6836325B13819CF38f030108255A5213491A725;
-address iDistroOpSepoliaAddress = 0x301C739CF6bfb6B47A74878BdEB13f92F13Ae5E7;
-    
+contract UpgradeGardenUnipool is Script {
     ProxyAdmin unipoolProxyAdmin;
-    ITransparentUpgradeableProxy unipoolProxy;
+    TransparentUpgradeableProxy unipoolProxy;
     UnipoolGIVpower implementation;
     UnipoolGIVpower givpower;
     IDistro iDistro;
 
     function run() public {
-        unipoolProxyAdmin = ProxyAdmin(proxyAdminOpSepoliaAddress);
-        iDistro = IDistro(iDistroOpSepoliaAddress);
-        unipoolProxy = ITransparentUpgradeableProxy(payable(unipoolProxyOpSepoliaAddress));
+        unipoolProxyAdmin = ProxyAdmin(address(0x91c5C402B0B514f2D09d84b03b6C9f17Bd689e2D));
```

Comment:
we should consider using env variables or store the addresses as named variables so they can be identified. it is hard to know which chain these addresses are set to - alternatively a constants file would help us keep track of things like proxy admins and proxy addresses